qtawesome/iconic_font.py

```diff
@@ -126,14 +126,20 @@ def load_font(self, prefix, ttf_filename, charmap_filename, directory=None):
         directory: str or None, optional
             directory for font and charmap files
         """
+
+        def hook(o):
+            r={}
+            for k,v in o.iteritems():
```

Comment:
Is this python3 compatible?

We are using [six](https://pythonhosted.org/six/#six.iteritems) so maybe you can adjust so that it is Py2/3 compatible as well?